src/modules/Layout/Table/Table.scss

```diff
@@ -0,0 +1,57 @@
+@import 'index.scss';
+
+.j-table {
+  width: 100%;
+  text-align: left;
+
+  tbody {
+    border: $hf-border;
+  }
+
+  &__row {
+    min-height: 70px;
+    background-color: $hf-color-white;
+
+    &:last-child {
+
+      .j-table__td {
+        padding-bottom: 70px;
+      }
+    }
+  }
+
+  &__th {
+    padding-top: $hf-padding-cell;
+    padding-bottom: $hf-padding-cell;
+    color: $hf-color-gray-medium;
+    font-weight: 700;
```

Comment:
there should be a variable for this and normally the maximum font weight we use is 500.